esp/templates/program/modules/teachercheckinmodule/missingteachers.html

```diff
@@ -158,17 +158,22 @@
         <td class="checkin-column">
             {% if teacher.id not in arrived %}
             <span class="message"></span>
-            <input id="checkin_{{ teacher.username }}" name="{{ teacher.last_name }}"
+            <input id="checkin_{{ teacher.username }}" name="{{ section.parent_class.emailcode }}: {{ section.name }}"
                    class="button checkin" type="button" value="Check In" {% if when %}disabled="disabled"{% endif %} /><span />
             {% else %}
             Already checked-in
             {% endif %}
         </td>
         <td class="phone"> {{ teacher.phone }} </td>
+        <td class="text-teacher">
+            <span class="message"></span>
+            <input id="text_{{ teacher.username }}" name="{{ section.id }}"
+                   class="button text" type="button" value="Text Teacher"
+                   {% if text_not_config %} disabled{% endif %}/>
```

Comment:
Maybe this should have title-text if it's disabled, saying to configure texting?